apis/sgv2-graphqlapi/src/main/resources/application.yaml

```diff
@@ -0,0 +1,49 @@
+stargate:
+  graphql:
+    # Whether to default to the oldest keyspace when the user accesses /graphql.
+    # If this is disabled, /graphql throws an error, and the keyspace must be provided explicitly in
+    # the path, as in /graphql/ksName.
+    enable-default-keyspace: true
+
+    # Whether to expose the GraphQL Playground in-browser IDE at /playground.
+    enable-playground: true
+
+  metrics:
+    global-tags:
+      module: sgv2-graphqlapi
+    tenant-request-counter:
+      enabled: ${stargate.multi-tenancy.enabled}
+
+  multi-tenancy:
+    enabled: false
+
+quarkus:
+
+  http:
```

Comment:
It uses the defaults defined by `AuthConfig`.

Reply:
Yea but you are not making Quarkus have any paths authenticated, everything can be accessed, is this the way to go? This is how it's done in the docs:

```
  # HTTP settings
  http:
    # TODO update both port and non-app paths after discussions
    port: 8180
    non-application-root-path: stargate

    # every /v2 path is authenticated by default
    # adapt if changing the authentication mechanism
    auth:
      proactive: false
      permission:
        default:
          paths: /v2/*
          policy: authenticated
```

This means, at least for the default `AuthConfig`, if the token header is not there, the request to that path would fail immediately, not even reaching the graphql, nor the bridge.. You are effectively saving at least one bridge call and all the processing in the graphql app itself.. 

For example, `/graphql/{keyspaceName}` seems like something that must have a token supplied.. Thus, I would expect all the endpoints that depend on having the token to be in that `paths` list.

Reply:
And one more thing here, if you go with the proposal, then there is additional work needed to be done with the metrics.. There is a strange bug in Quarkus that does not correctly report paths in metrics once auth fails: https://github.com/quarkusio/quarkus/issues/24938

In order to overcome this, you need to define `match-patterns` in order to not have variable paths in the metrics like:

```
  # built-in micrometer properties
  micrometer:
    binder:
      http-server:
        # ignore all non-application uris, as well as the custom set
        suppress-non-application-uris: true
        ignore-patterns: /,/metrics,/swagger-ui.*,.*\.html

        # due to the https://github.com/quarkusio/quarkus/issues/24938
        # we need to define uri templating on our own for now
        # note that order is important
        match-patterns: |
          /v2/namespaces/[\\w]+/collections=/v2/namespaces/{namespace:\\w+}/collections,
          /v2/namespaces/[\\w]+/collections/[\\w]+=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+},
          /v2/namespaces/[\\w]+/collections/[\\w]+/batch=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/batch,
          /v2/namespaces/[\\w]+/collections/[\\w]+/json-schema=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/json-schema,
          /v2/namespaces/[\\w]+/collections/[\\w]+/upgrade=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/upgrade,
          /v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/.*/function=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id}/{document-path:.*}/function,
          /v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/.*=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id}/{document-path:.*},
          /v2/namespaces/[\\w]+/collections/[\\w]+/.*=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id},
          /v2/schemas/namespaces/[\\w]+=/v2/schemas/namespaces/{namespace:\\w+}
```

For you this should be simple, as you only have two endpoints with path parameters:

* `/graphql/{keyspaceName}`
* `/graphql-files/keyspace/{keyspaceName}.graphql`